content/projects/tdd/string-calculator-part-1/_index.md

```diff
@@ -24,27 +24,32 @@ Please test your code using jasmine.
 Your directory structure should look like this.
 
 ```
-    >node_modules    <---- make sure this is in your .gitignore
-    >spec
-        > support
-            - jasmine.json
-        - string_calculator_spec.js
-    >src
-        - string_calculator.js
-    - package.json
+├── spec
+|   ├── support
+|   |   └── jasmine.json
+|   └── string_calculator_spec.js
+├── src
+|   └── string_calculator.js
+└── package.json
+```
+
+**Note**: Please export your `add` function using the following syntax at the end of the code:
+
+```
+module.exports = {functionName}
 ```
 
 ### Python
 
 Your project is expected to be completed using pytest. You are expected to follow industry best practices in all things. This means that you need to have a directory structure that is in good shape. Please name your files and folders like this:
 
-```
-├── string_calculator   the package under test
-│   └── calculator.py
-├── requirements.txt    installation requirements
-├── setup.py            installation script for the package under test
-└── tests               all package tests go in this directory
-    └── test_calculator.py
+```py
+├── string_calculator   # the package under test
+│   └── string_calculator.py
```

Comment:
If you rename this then the current string calc automaker config will stop working. It'll need to be renamed in both locations.

It would be good to check that part 2 is consistent as well.

Reply:
I updated the automarker project. I will open a separate PR for part 2 if updates are needed